source/relations/sec-properties.ptx

```diff
@@ -42,7 +42,7 @@
             </ul>
         </p>
         <p>
-            Sage is open-source and many people work on the software for free, when they can find the available time. While Sage has many built-in methods, in some cases, we may need to define our own functions to check for certain properties. When you see the <c>def</c> keyword, it means that we are defining a function. The function is a block of code that only runs when it is called. If you would like to reuse the functions defined in this section, you can copy and paste the function definitions into your own Sage worksheet. For methods built into Sage, you can use them directly, without including the definitions. If you feel inspired, try <url href="https://doc.sagemath.org/html/en/faq/faq-contribute.html">contributing</url> to the official Sage project.
+            Sage is open-source and people work on the software for free. While Sage has many built-in methods, in some cases, we may need to define our own functions to check for certain properties. When you see the <c>def</c> keyword, we are defining a function. The function is a block of code that only runs when it is called. If you want to reuse the functions defined in this section, you can copy and paste the function definitions into your own Sage worksheet. You can use methods built into Sage directly without including the definitions. If you feel inspired, try <url href="https://doc.sagemath.org/html/en/faq/faq-contribute.html">contributing</url> to the official Sage project.
```

Comment:
I feel that now the first and last sentence are disconnected. Why are we saying here that Sage is open-source? I think that the reason is to say that there are functions already defined and others that can be defined by you in the future if you want. Instead of just change the particular words pointed out by the reviewer, maybe think the whole paragraph again to say what we really mean?